polar_plotter_AREA.py

```diff
@@ -0,0 +1,180 @@
+## File name: polar_plotter.py
+## Author: Alex Machtay (machtay.1@osu.edu)
+## Date: 07/19/21
+
+## AREA-specific Edits: Bryan Reynolds (reynolds.886@osu.edu)
+## Date: 11/2/22
+
+## Purpose:
+##	This script is designed to read in the results from XF and plot the gain patterns.
+##	It will plot each of the antennas given (in a range) at a specific (given) frequency.
+##	These will show the gain at zenih angles, since the patterns are azimuthally symmetric
+#
+#
+#
+## Instructions:
+## 		To run, give the following arguments
+##			source directory, destination directory, frequency number, initial antenna, final antenna 
+#
+## 		Example:
+##			python polar_plotter.py Length_Tweak Length_Tweak 60 118 127
+## 				This will plot the antennas for the length tweak from number 118-127
+##				(which is all of them) at the highest frequency number (1067 MHz) and
+##				will place the image inside the Length_Tweak directory.
+#		
+#		For AREA:
+#			python polar_plotter.py ~/AREA/GA/Runs/$RUNNAME/#GEN_CNT ~/AREA/GA/Runs/$RUNNAME/$GEN_CNT FREQNUM(1-60) 0 24
+## Imports
+import numpy as np
+import matplotlib.pyplot as plt
+import pandas as pd
+import argparse
+import csv
+import matplotlib.cm as cm
+#
+#
+#
+## Arguments
+parser = argparse.ArgumentParser()
+parser.add_argument("source1", help="Name of source folder from home directory", type=str)
+#parser.add_argument("source2", help="Name of source folder from home directory", type=str)
+parser.add_argument("destination", help="Name of destination folder from home directory", type=str)
+parser.add_argument("freq_num", help="Frequency number (1-60) to plot", type=int)
+parser.add_argument("initial_antenna", help="First antenna number to start reading (ex: 65)", type=int)
+parser.add_argument("final_antenna", help="Last antenna number to start reading (ex: 69)", type=int)
+g=parser.parse_args()
+#
+#
+#
+## Loop over files
+# Declare list for each file's gain list
+gains = []
+realized_gains = []
+
+#Define ranges of lines to be read for azimuthal gains at the given freq_num
+#There are 2668 lines for each frequency block (including headers)
+#And azimuthal gain values are lines 4-40 counting from the header of each freq
+#So read lines between (2668*freq_num)+4 and (2668*freq_num)+40
+azimuthGainStart = ((g.freq_num-1)*2668)+4
+azimuthGainEnd = ((g.freq_num-1)*2668)+40
+
+#print(azimuthGainStart)
+#print(azimuthGainEnd)
+
+for i in range(g.initial_antenna, g.final_antenna+1):
+	# Gains
+	## Open the file to read
+	with open(g.source1 + "/" + "child" + "_" + str(i) + ".txt", "r") as f:
+		## Read in the file
+		uan_read = csv.reader(f, delimiter = ' ')
```

Comment:
```suggestion
		uan_read = csv.reader(f, delimiter=' ')
```
There should be no spaces around the `=` for named arguments (but for regular assignments you should add them).